source/Map.svelte

```diff
@@ -506,9 +508,15 @@
       }
     });
   });
+
+  const closeSettingsForSmallDevices = () => {
+    if (innerWidth <= 1100) {
```

Comment:
This confused me for a second;

```suggestion
    if (window.innerWidth <= 1100) {
```